src/viur/core/utils.py

```diff
@@ -272,6 +273,20 @@ def normalizeKey(key: Union[None, 'db.KeyClass']) -> Union[None, 'db.KeyClass']:
     return db.Key(key.kind, key.id_or_name, parent=parent)
 
 
+def parse_bool(value: Any, truthy_values: typing.Iterable = ("true", "yes", "1")) -> bool:
```

Comment:
This function is nice, but I would like to have it together with other parsing functions [like parse_int and parse_float](https://github.com/viur-framework/logics/blob/main/logics-py/logics/value.py)  currently implemented in Logics in a separate module. What do you think?

Reply:
Do you want to integrate logics in the core or put the methods together with my boolean parse method in a new `parse` module (py-file)?

Reply:
:-D Surely I would be glad to integrate Logics as well into viur-core. Anyway, this is another huge step we're moving into the future.

Can we make this pull request as a separate PR after https://github.com/viur-framework/viur-core/pull/831 has been merged?